subsys/caf/modules/sensor_sampler.c

```diff
@@ -400,6 +484,67 @@ static bool event_handler(const struct event_header *eh)
 		return false;
 	}
 
+	if (IS_ENABLED(CONFIG_CAF_SENSOR_SAMPLER_PM) && is_power_down_event(eh)) {
+		for (size_t i = 0; i < ARRAY_SIZE(sensor_data); i++) {
```

Comment:
seems long. can we move the code to
`handle_power_down_event()`
`handle_wake_up_event()`

Reply:
Ok